client/js/widgets/pile.js

```diff
@@ -198,6 +209,17 @@ class Pile extends Widget {
     this.handle.textContent = text === null ? this.childCount : text;
   }
 
+  updateVisibleChildren() {
+    for (let child of this.visibleChildren) {
```

Comment:
why `let` here but `const` in the next loop?

Reply:
Now const here too :)